src/languages/en.ts

```diff
@@ -3217,6 +3217,25 @@ export default {
     subscription: {
         mobileReducedFunctionalityMessage: 'You can’t make changes to your subscription in the mobile app.',
         billingBanner: {
+            outdatedInfo: 'Your payment info is outdated',
```

Comment:
It is difficult to know if this is a title/description key.  Can we append `Description` to all the keys that are description?